### ISSUE TITLE
Ability to export RIS files for import into reference managers or distiller

### ISSUE THREAD

**Issue by ursjoss** [ursjoss/scipamato#118]
- http://v2dis-help.evidencepartners.com/1/en/topic/uploading-references
- https://github.com/fastluca/JRis

If possible also export associated PDFs.

**Comment by ursjoss** [comment]
```
TY  - JOUR
ID  - 29144419
T1  - Air Pollution and Dispensed Medications for Asthma, and Possible Effect Modifiers Related to Mental Health and Socio-Economy: A Longitudinal Cohort Study of Swedish Children and Adolescents.
A1  - Oudin A
AU  - Bråbäck L
AU  - Oudin Åström D
AU  - Forsberg B
PY  - 2017
SP  - pii: E1392
EP  - 
JO  - Int J Environ Res Public Health
VL  - 14
IS  - 11
N2  - ...
L1  - ...
M1  - Registerbasierte Kohortenstudie in Schweden zur Untersuchung, ob Asthma mit der langfristigen NO2-Belastung zusammenhänge und ob diese Beziehung durch die geistige Gesundheit und Stress beeinflusst wird.
ER  - 
```

where
- SP: start page
- EP: end page
- N2: abstract
- M1: goal
- L1: potentially link to pdf

**Comment by ursjoss** [comment]
https://github.com/fastluca/JRis/issues/4

**Comment by ursjoss** [comment]
Closed via #129.
